whisper-playground/Containerfile

```diff
@@ -0,0 +1,14 @@
+FROM quay.io/centos/centos:stream8
+WORKDIR /app
+RUN dnf install -y make gcc gcc-c++ git && \
+  rm -rf /var/cache/dnf/*
+RUN git clone https://github.com/hemajv/whisper.cpp.git . && \
```

Comment:
Do we need a fork ?

Reply:
The reason I used my fork was so that it would be static and not have any issues if the upstream repo had minor changes that could break the image. We could build from the upstream repo as well, but need to monitor for any changes that occur.

Reply:
we can also use the 1.5.4 tag to pin the upstream repo

Reply:
@sallyom sure will do that 👍

Reply:
@hemajv ptal here: https://github.com/hemajv/locallm/pull/1 I think this works well, wdyt?

Reply:
@sallyom tysm, going to test this out and merge in a bit 👍

Reply:
updated the Containerfile with Sally's suggested changes